addons/kblib/values.yaml

```diff
@@ -11,6 +11,10 @@ terminationPolicy: Delete
 ##
 availabilityPolicy: node
 
+## @param affinityNodeLabels
+##
+affinityNodeLabels: {}
```

Comment:
affinityNodeLabels is expected to be a map. could you pls give some more exmaples to show how to set the values.

Reply:
Fixed in f8e85c0cb0314da89fc9739599eb412d00145aef